### PR TITLE
odbc_pcconnect for PHP >=5.6

### PR DESCRIPTION
**Pull request by rajasharma27**
using odbc_pconnect for latest php version

### CODE REVIEW

File: php/elFinderVolumeMsSQL.class.php [hunk 1]
```diff
@@ -98,12 +98,18 @@ protected function init() {
 			return false;
 		}
 
+		try{
+			$this->conn = odbc_pconnect("DRIVER={SQL Server};Server=".$this->options['host'].";ExtendedAnsiSQL=1;Database=".$this->options['db'].";UID=".$this->options['user'].";PWD=".$this->options['pass'],"","");
+			// odbc_pcconnect php >=5.6
+  			}catch(Exception $e) {
```
Comment (troex) [inline php/elFinderVolumeMsSQL.class.php]: `catch(Exception $e)` is a bad practice, please catch exact error.

**Comment by rajasharma27** [comment]
please suggest, can be try three connection 

$this->conn = @odbc_pconnect("DRIVER={SQL Server};Server=".$this->options['host'].";ExtendedAnsiSQL=1;Database=".$this->options['db'].";UID=".$this->options['user'].";PWD=".$this->options['pass'],"","");   // odbc_pcconnect php >=5.6
		
		if (!$this->conn){
		$this->conn = @odbc_connect("DRIVER={SQL Server};Server=".$this->options['host'].";ExtendedAnsiSQL=1;Database=".$this->options['db'].";UID=".$this->options['user'].";PWD=".$this->options['pass'],"","");   // odbc_pcconnect php >=5.6
		}
		if (!$this->conn){	
		$this->conn = @odbc_connect("Driver={SQL Server};Server=".$this->options['host'].";Database=$this->options['db'];", $this->options['user'], $this->options['pass']);   //odbc_connect php <=5.4
		}
		
		if (!$this->conn){	
			return false;
		}

File: php/elFinderVolumeMsSQL.class.php [hunk 2]
```diff
@@ -98,12 +98,19 @@ protected function init() {
 			return false;
 		}
 
-
-		$this->conn = odbc_connect("Driver={SQL Server};Server=".$this->options['host'].";Database=$this->options['db'];", $this->options['user'], $this->options['pass']);
-
-		if (odbc_errormsg($this->conn)) {
+		$this->conn = @odbc_pconnect("DRIVER={SQL Server};Server=".$this->options['host'].";ExtendedAnsiSQL=1;Database=".$this->options['db'].";UID=".$this->options['user'].";PWD=".$this->options['pass'],"","");   // odbc_pcconnect php >=5.6
```
Comment (nao-pon) [inline php/elFinderVolumeMsSQL.class.php]: Why use `odbc_pconnect`? Is it correct?

**Comment by nao-pon** [comment]
There is a regrettable, but we drop the MsSQL driver. Because it can not be maintained to continue because I am not familiar with MsSQL.

So you can listed in the README as a third-party volume driver, please PR to README.md.

**Comment by troex** [comment]
@nao-pon I think we should extract MsSQL driver into separate repository, and put in readme that it needs maintainer, I'll create issue

**Comment by rajasharma27** [comment]
sir,
ok
thanks

**Pull request closed without merging**